cmd/cli/status.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright © 2023 SIL International
+*/
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/silinternational/tfc-ops/lib"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var statusCmd = &cobra.Command{
+	Use:   "status",
+	Short: "Read the current status of the IdP",
+	Long:  `Read the current status of the IdP. Does not modify any infrastructure.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		status()
+	},
+}
+
+func setupStatusCmd(rootCmd *cobra.Command) {
+	rootCmd.AddCommand(statusCmd)
+}
+
+func status() {
+	idp := viper.GetString("idp")
+	if idp == "" {
+		log.Fatalln("no IdP key is set, use --idp to set the IdP key")
+	}
+
+	org := viper.GetString("org")
+	if org == "" {
+		log.Fatalln("no org is set, use --org to set the Terraform Cloud org")
+	}
+
+	token := viper.GetString("token")
+	if token == "" {
+		log.Fatalln("no token is set, use --token to set the Terraform Cloud API token")
+	}
+	lib.SetToken(token)
+
+	workspaceName := fmt.Sprintf("idp-%s-prod-000-core", idp)
+	failoverActive, err := lib.GetWorkspaceVar(org, workspaceName, "aws_failover_active")
+	if err != nil {
+		return
```

Comment:
Shouldn't we say something about the error here?

Reply:
Yes, thanks for catching that.